classes/Infrastructure/Auth/SentinelAccountManagement.php

```diff
@@ -88,19 +88,12 @@ public function create(string $email, string $password, array $data = []): UserI
         throw new UserExistsException();
     }
 
-    /**
-     * @param string $email
-     *
-     * @throws UserNotFoundException
-     *
-     * @return bool
-     */
```

Comment:
Removing the docblock here as it is added to the `AccountManagement` interface in #814.